synapse/lib/layer.py

```diff
@@ -419,6 +426,8 @@ def _liftUtf8Regx(self, liftby, valu):
             if buid == lastbuid:
                 continue
 
+            await asyncio.sleep(0)
```

Comment:
Is it possible for buidByPref to spin in the if statement prior to this line?

Reply:
Only up to the max number of layer rows for a node.  I opted for this as an "only yield every N iters" mechanism.  Ultimately, this boundary is the same as the "get all the props for this buid" which similarly doesn't yield.